dlp/api/Snippets.Tests/RedactSensitiveDataFromImageUsingDefaultInfoTypesTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.IO;
+using Xunit;
+
+namespace GoogleCloudSamples
+{
+    public class RedactSensitiveDataFromImageUsingDefaultInfoTypesTests : IClassFixture<DlpTestFixture>
+    {
+        private DlpTestFixture _fixture;
+        public RedactSensitiveDataFromImageUsingDefaultInfoTypesTests(DlpTestFixture fixture) => _fixture = fixture;
+
+        [Fact]
+        public void TestRedactSensitiveData()
+        {
+            var input = Path.Combine(_fixture.ResourcePath, "test_redact_sensitive_data.png");
+            var output = Path.Combine(_fixture.ResourcePath, "redacted.png");
```

Comment:
Please use unique output file names to avoid flakiness of tests (not sure if it can cause problems, but I feel it is better)!

Reply:
Done.